### PR TITLE
Add ajuna network

### PR DESCRIPTION
**Pull request by cowboy-bebug**
This PR adds app configuration for Ajuna Network.
Thanks.

### CODE REVIEW

File: packages/apps-config/src/api/spec/ajuna.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+/* eslint-disable sort-keys */
+const definitions: OverrideBundleDefinition = {
+  types: [
+    {
+      // on all versions
+      minmax: [0, undefined],
+      types: {
+        Address: 'MultiAddress',
+        LookupSource: 'MultiAddress',
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/ajuna.ts:14]: Linting fails here, see https://github.com/polkadot-js/apps/runs/5180186370?check_suite_focus=true#step:3:696

File: packages/apps-config/src/ui/colors.ts [hunk 2]
```diff
@@ -216,6 +218,7 @@ export const chainColors: Record<string, string> = Object.entries({
   'acala mandala pc2': chainRoccoMandala,
   'Aleph Zero': chainAleph,
   'Aleph Zero Testnet': chainAleph,
+  Ajuna: chainAjuna,
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:221]: Sort order, https://github.com/polkadot-js/apps/runs/5180186370?check_suite_focus=true#step:3:699

**Review by jacogr — CHANGES_REQUESTED** [review]
CI fails on some linting checks, apart from that looks good.

File: packages/apps-config/src/api/spec/ajuna.ts (hunk 1 above)
Comment (cowboy-bebug) [reply packages/apps-config/src/api/spec/ajuna.ts]: Thanks!

File: packages/apps-config/src/ui/colors.ts (hunk 2 above)
Comment (cowboy-bebug) [reply packages/apps-config/src/ui/colors.ts]: Thanks!

**Comment by cowboy-bebug** [comment]
Thanks 🙏 could you re-run the workflows?

File: packages/apps-config/src/endpoints/testing.ts [hunk 3]
```diff
@@ -24,6 +24,13 @@ export function createTesting (t: TFunction, firstOnly: boolean, withSort: boole
         'Aleph Zero Foundation': 'wss://ws.test.azero.dev'
       }
     },
+    {
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:27]: Sort alphabetically, see https://github.com/polkadot-js/apps/runs/5195053157?check_suite_focus=true#step:3:759

**Review by jacogr — CHANGES_REQUESTED** [review]

File: packages/apps-config/src/endpoints/testing.ts (hunk 3 above)
Comment (cowboy-bebug) [reply packages/apps-config/src/endpoints/testing.ts:27]: Thanks

**Review by jacogr — APPROVED** [review]

**Pull request merged**